src/db/couch-db.js

```diff
@@ -0,0 +1,186 @@
+'use strict'
+
+const dbDestroy = (dbClient, collectionName) => {
```

Comment:
Maybe this could use Bluebird `promisifyAll` in order to be able to use async/await ?

Reply:
I dont see the need for this one to be async.  Am I missing something from a vanilla promise perspective?